### ISSUE TITLE
E-mail notiffications

### ISSUE THREAD

**Issue by pax0707** [Tautulli/Tautulli#691]
1st one gets sent, rest don't.
Log:

```
Timestamp   Level   Message
2016-05-07 00:59:48 ERROR   Uncaught exception: Traceback (most recent call last):
File "D:\Plex\PlexPy\plexpy\logger.py", line 288, in new_run
old_run(*args, **kwargs)
File "C:\Python27\lib\threading.py", line 754, in run
self.__target(*self.__args, **self.__kwargs)
File "D:\Plex\PlexPy\plexpy\notification_handler.py", line 99, in notify
agent_id=agent['id'])
File "D:\Plex\PlexPy\plexpy\notification_handler.py", line 577, in build_notify_text
+ rating_key)
TypeError: cannot concatenate 'str' and 'int' objects
2016-05-07 00:59:47 DEBUG   PlexPy Monitor :: Session 62 has been paused.
2016-05-07 00:58:50 INFO    PlexPy Notifiers :: Email notification sent.
2016-05-07 00:58:47 DEBUG   PlexPy Monitor :: Session 62 has started.
2016-05-07 00:57:17 DEBUG   User 'pax0707' logged into PlexPy.
2016-05-07 00:56:47 INFO    Starting PlexPy web server on http://0.0.0.0:8181/
2016-05-07 00:56:47 INFO    Web server authentication is enabled, username is 'pax0707'
2016-05-07 00:56:47 INFO    Scheduled background task: Backup PlexPy database
2016-05-07 00:56:47 INFO    Scheduled background task: Refresh libraries list
2016-05-07 00:56:47 INFO    Scheduled background task: Refresh users list
2016-05-07 00:56:47 INFO    Scheduled background task: Check for active sessions
2016-05-07 00:56:47 INFO    Scheduled background task: Check for Plex remote access
2016-05-07 00:56:47 INFO    Scheduled background task: Check for Plex updates
2016-05-07 00:56:47 INFO    Scheduled background task: Check for recently added items
2016-05-07 00:56:47 INFO    Scheduled background task: Refresh Plex server name
2016-05-07 00:56:47 INFO    Scheduled background task: Refresh Plex server URLs
2016-05-07 00:56:47 INFO    Scheduled background task: Check GitHub for updates
2016-05-07 00:56:47 INFO    PlexPy Pmsconnect :: Libraries list refreshed.
2016-05-07 00:56:45 INFO    PlexPy Pmsconnect :: Requesting libraries list refresh...
2016-05-07 00:56:45 INFO    PlexPy PlexTV :: Users list refreshed.
2016-05-07 00:56:44 INFO    PlexPy PlexTV :: Requesting users list refresh...
2016-05-07 00:56:44 INFO    PlexPy Pmsconnect :: Requesting name from server...
2016-05-07 00:56:44 INFO    PlexPy PlexTV :: Using user-defined URL.
2016-05-07 00:56:44 INFO    Writing configuration to file
2016-05-07 00:56:44 INFO    Writing configuration to file
2016-05-07 00:56:44 INFO    PlexPy PlexTV :: Requesting URLs for server...
2016-05-07 00:56:44 INFO    PlexPy is up to date
2016-05-07 00:56:44 DEBUG   Latest version is 81f9f52353969718178d216f5bc6a8990c3bf39f
2016-05-07 00:56:42 DEBUG   Requesting URL via GET method: https://api.github.com/repos/drzoidberg33/plexpy/commits/dev
2016-05-07 00:56:42 INFO    Retrieving latest version information from GitHub
2016-05-07 00:56:42 DEBUG   Git output: dev
2016-05-07 00:56:42 DEBUG   Trying to execute: "git rev-parse --abbrev-ref HEAD" with shell in D:\Plex\PlexPy
2016-05-07 00:56:42 DEBUG   Git output: 81f9f52353969718178d216f5bc6a8990c3bf39f
2016-05-07 00:56:42 DEBUG   Trying to execute: "git rev-parse HEAD" with shell in D:\Plex\PlexPy
2016-05-07 00:56:42 INFO    Checking to see if the database has all tables....
```

**Comment by JonnyWong16** [comment]
`dev` bug. Thanks.

**Comment by JonnyWong16** [comment]
Fixed e50c77d8c62138a79c27cfcc5b7989dfc2f341f2
